Misc/HISTORY

```diff
@@ -966,7 +966,7 @@ Core and Builtins
   returned NotImplemented.  Original patch by Martin Panter.
 
 - Issue #23321: Fixed a crash in str.decode() when error handler returned
-  replacment string longer than mailformed input data.
+  replacement string longer than mailformed input data.
```

Comment:
Bonus points for fixing malformed _mailformed_ spelling :)

Reply:
oh yes, thank you. ;-)